test/spec/EditorCommandHandlers-test.js

```diff
@@ -466,9 +489,314 @@ define(function (require, exports, module) {
 
             describe("with multiple selections", function () {
                 it("should toggle comments on separate lines with cursor selections", function () {
+
+                   myEditor.setSelections([{start: {line: 1, ch: 4}, end: {line: 1, ch: 4}},
+                                        {start: {line: 3, ch: 4}, end: {line: 3, ch: 4}}]);
+
+                    var lines = defaultContent.split("\n");
+                    lines[1] = "    //function bar() {";
+                    lines[3] = "        //a();";
+                    var expectedText = lines.join("\n");
+
+                    testToggleLine(expectedText, [{start: {line: 1, ch: 6}, end: {line: 1, ch: 6}, primary: false, reversed: false},
+                                                  {start: {line: 3, ch : 4 }, end: {line: 3, ch: 4 }, reversed: false, primary: true}]);
+                });
+
+                it("should toggle comments on separate lines with range selections", function () {
+
+                    myEditor.setSelections([{start: {line: 1, ch: 4}, end: {line: 1, ch: 6}},
+                                        {start: {line: 3, ch: 4}, end: {line: 3, ch: 6}}]);
+
+                    var lines = defaultContent.split("\n");
+                    lines[1] = "    //function bar() {";
+                    lines[3] = "        //a();";
+                    var expectedText = lines.join("\n");
+
+                    testToggleLine(expectedText, [{ start: {line: 1, ch: 6 }, end: {line: 1, ch: 8 }, reversed: false, primary: false },
```

Comment:
Stylenit: the test object has different spacing than the other `testToggleLine`'s (no space before, space after `,` and `;`